### PR TITLE
Make a test file for PropertiesFileOverridesReporter class

### PR DESCRIPTION
**Pull request by FeimeiChen**
# Ticket Link

[Ticket 1504](https://uhawaii.atlassian.net/browse/GROUPINGS-1504)

# List of squashed commits

- Commit 1: Make a test file for PropertiesFileOverridesReporter class
- Commit 2: Edit .gitignore file

# Test Checklist

- [x] Unit Tests Passed:
- [x] Jasmine Tests Passed:
- [x] General Visual Inspection:

### CODE REVIEW

File: src/test/java/edu/hawaii/its/groupings/configuration/PropertiesFileOverridesReporterTest.java
Comment (JorWo) [inline src/test/java/edu/hawaii/its/groupings/configuration/PropertiesFileOverridesReporterTest.java:1]: Can you press `CTRL + ALT + l` (not i) in IntelliJ for this file to autoformat it with our code style. Make sure the there is an extra space at the end of the file.

File: src/test/java/edu/hawaii/its/groupings/configuration/PropertiesFileOverrideReporterTest.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package edu.hawaii.its.groupings.configuration;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
```
Comment (fduckart) [inline src/test/java/edu/hawaii/its/groupings/configuration/PropertiesFileOverrideReporterTest.java:9]: Can you use the hamcrest assertThat here instead? We've been using that everywhere else.

Comment (FeimeiChen) [reply src/test/java/edu/hawaii/its/groupings/configuration/PropertiesFileOverrideReporterTest.java]: done.

**Review by JorWo — APPROVED** [review]
Looks great!

**Comment by JorWo** [comment]
One more thing, could you check if the JaCoCo test coverage is 100% for the test you have created? Thanks.

**Comment by FeimeiChen** [comment]
> One more thing, could you check if the JaCoCo test coverage is 100% for the test you have created? Thanks.
image

**Pull request merged**